tests/collaborative/collaborative_sheet_manipulations.test.ts

```diff
@@ -570,6 +570,7 @@ describe("Collaborative Sheet manipulation", () => {
       dataSets: ["A1:A3", "F1:F3"],
       labelRange: "F3",
       title: "chart title",
+      titleType: "string",
```

Comment:
should be a reference, and test that it's correclty modified in the test of this file